### DISCUSSION TITLE
Give segmentation masks for torch version

### DISCUSSION THREAD

**Discussion by d6222g** [voxelmorph/voxelmorph#608] [Q&A]
Is it possible to train the model with the segmentation masks (e.g. for abdominal CT kidney..) with the pytorch version ? Has anyone tried semi-supervised torch code?
